Code/Tools/AssetProcessor/native/tests/assetmanager/AssetProcessorManagerTest.cpp

```diff
@@ -4231,34 +4241,23 @@ TEST_F(LockedFileTest, DeleteFile_LockedProduct_DeletesWhenReleased)
     // If we can, it means the OS running this test doesn't lock open files so there's nothing to test
     if (!AZ::IO::SystemFile::Delete(productPath.toUtf8().constData()))
     {
-        AZStd::atomic_bool threadStarted = false;
+        m_deleteCounter = 0;
```

Comment:
I think we should reset this to 0 at the top of test, outside of any if blocks.
I more thinking about the `--gtest_repeat` workflow.

Reply:
--gtest_repeat=10 passed all 10 times.  I put this here just to be extra sure but I'm pretty certain its not needed.  The test class is always destroyed at the end of a test and created again for the next.  We only run into problems when working with static data as far as I've seen.